### PR TITLE
Translation backport

### PR DESCRIPTION
**Pull request by teolemon**

### CODE REVIEW

File: po/common/fr.po [hunk 1]
```diff
@@ -4919,12 +4903,86 @@ msgstr "Vos préférences alimentaires sont conservées dans votre navigateur et
...
+msgstr "écoscore"
+
+msgctxt "packaging_parts"
+msgid "Packaging parts"
+msgstr "Parties de l'emballage"
+
+# Number of packaging parts
+msgctxt "packaging_number"
+msgid "Number"
+msgstr "Numéro"
```
Comment (stephanegigandet) [inline po/common/fr.po:4931]: Number as in a number of packaging part, so not "Numéro" but "Nombre".

File: po/common/fr.po [hunk 2]
```diff
@@ -4919,12 +4903,86 @@ msgstr "Vos préférences alimentaires sont conservées dans votre navigateur et
...
+
+# Used to indicate if a product is ok for a specific diet. It needs to be very short, keep "OK" if it's commonly used in the language
+msgctxt "match_no"
+msgid "Not OK"
+msgstr "Non"
+
+# Used to indicate if a product is ok for a specific diet. It needs to be very short, keep "OK" if it's commonly used in the language
+msgctxt "match_unknown"
+msgid "Maybe OK"
+msgstr "Peut-être"
```
Comment (teolemon) [inline po/common/fr.po:4983]:
```suggestion
msgstr "Peut-être OK"
```

File: po/common/fr.po [hunk 3]
```diff
@@ -4919,12 +4903,86 @@ msgstr "Vos préférences alimentaires sont conservées dans votre navigateur et
...
+
+# Used to indicate if a product is ok for a specific diet. It needs to be very short, keep "OK" if it's commonly used in the language
+msgctxt "match_yes"
+msgid "OK"
+msgstr "OK"
+
+# Used to indicate if a product is ok for a specific diet. It needs to be very short, keep "OK" if it's commonly used in the language
+msgctxt "match_no"
+msgid "Not OK"
+msgstr "Non"
```
Comment (teolemon) [inline po/common/fr.po:4978]:
```suggestion
msgstr "Non OK"
```

File: po/common/fr.po (hunk 1 above)
Comment (teolemon) [inline po/common/fr.po:4931]:
```suggestion
msgstr "Nombre"
```

**Review by stephanegigandet — APPROVED** [review]

**Pull request merged**